Client/Ecosia/UI/MultiplyImpact/MultiplyImpact.swift

```diff
@@ -64,12 +64,16 @@ final class MultiplyImpact: UIViewController, NotificationThemeable {
     private weak var topBackground: UIView?
     private weak var forestOverlay: UIView?
     private weak var waves: UIImageView?
-    private weak var card: UIView?
-    private weak var cardIcon: UIImageView?
-    private weak var cardTitle: UILabel?
-    private weak var cardTreeCount: UILabel?
-    private weak var cardTreeIcon: UIImageView?
     private weak var yourInvites: UILabel?
+    private lazy var referralImpactRowView: NTPImpactRowView = {
+        let info = ClimateImpactInfo.referral(value: User.shared.referrals.impact,
+                                              invites: User.shared.referrals.count)
+        let view = NTPImpactRowView(info: info)
+        view.info = info // Needed to force info setup after init
```

Comment:
Interested to get more insights on this one. Is it strictly needed?
I tried to put a breakpoint in the `didSet` of that `var info: ClimateImpactInfo` and seems like it gets there even without re-assigning the value here.

Reply:
It unfortunately is, this basically resulted from the fact that I prepared the view to automatically refresh every time the info changes. It initially was not even passed on init (mimicking the behaviour of a cell that is configured afterwards), but I had to pass it solely to support the existence of the progress bar. 

You are right that `didSet` is in fact already called, but for some reason that is not the correct point in the lifecycle for it to properly update. `defer` is an interesting suggestion and should in fact work, I can try it since it seems less error prone 👍

Reply:
It also works with the `defer`, just updated it, thanks for the suggestion ❤️